### PR TITLE
[IMP] website: add new `s_faq_list` snippet

### PR DESCRIPTION
**Pull request by anso-odoo**
This commit introduces the new `s_faq_list` text snippet.

Part of task-4077427
task-4094399

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/snippets/s_faq_list.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                    <h6><strong>Use of cookies</strong></h6>
+                    <p>Website may use cookies to personalize and facilitate maximum navigation of the User by this site.</p>
+                </div>
+                <div class="col-lg-4 pt16 pb16">
+                    <h6><strong>Terms of service </strong></h6>
+                    <p>These terms of service ("Terms", "Agreement") are an agreement between the website ("Website operator", "us", "we" or "our") and you ("User", "you" or "your").</p>
+                </div>
+                <div class="col-lg-4 pt16 pb16">
+                    <h6><strong>Use of cookies </strong></h6>
+                    <p>Website may use cookies to personalize and facilitate maximum navigation of the User by this site.</p>
```
Comment (stefanorigano) [inline addons/website/views/snippets/s_faq_list.xml:30]:
```suggestion
                    <h6><strong>Who are our partners?</strong></h6>
                    <p>We collaborate with trusted, high-quality partners to bring you reliable and top-notch products and services.</p>
```

File: addons/website/views/snippets/s_faq_list.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                    <h6><strong>Links to other websites </strong></h6>
+                    <p>Although this Website may be linked to other websites, we are not, directly or indirectly, implying any approval</p>
+                </div>
+                <div class="col-lg-4 pt16 pb16">
+                    <h6><strong>Use of cookies</strong></h6>
+                    <p>Website may use cookies to personalize and facilitate maximum navigation of the User by this site.</p>
+                </div>
+                <div class="col-lg-4 pt16 pb16">
+                    <h6><strong>Terms of service </strong></h6>
+                    <p>These terms of service ("Terms", "Agreement") are an agreement between the website ("Website operator", "us", "we" or "our") and you ("User", "you" or "your").</p>
```
Comment (stefanorigano) [inline addons/website/views/snippets/s_faq_list.xml:26]:
```suggestion
                    <h6><strong>What support do we offer?</strong></h6>
                    <p>We provide 24/7 support through various channels, including live chat, email, and phone, to assist with any queries.</p>
```

File: addons/website/views/snippets/s_faq_list.xml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                    <h3>Need help?</h3>
+                    <p class="lead">In this section, you can address common questions efficiently.</p>
+                </div>
+                <div class="col-lg-4 pt16 pb16">
+                    <h6><strong>Terms of service </strong></h6>
+                    <p>These terms of service ("Terms", "Agreement") are an agreement between the website ("Website operator", "us", "we" or "our") and you ("User", "you" or "your").</p>
+                </div>
+                <div class="col-lg-4 pt16 pb16">
+                    <h6><strong>Links to other websites </strong></h6>
+                    <p>Although this Website may be linked to other websites, we are not, directly or indirectly, implying any approval</p>
```
Comment (stefanorigano) [inline addons/website/views/snippets/s_faq_list.xml:18]:
```suggestion
                    <h6><strong>How is your data secured?</strong></h6>
                    <p>Your data is protected by advanced encryption and security protocols, keeping your personal information safe.</p>
```

**Review by stefanorigano — CHANGES_REQUESTED** [review]
Neat 🙂

Just the demo text it's a bit suboptimal 'cause it doesn't really fit the scenario.
It's not where I'd expect to see terms of service, for example, and the content of the last 3 columns is copied for the previous ones.

image


Some proposals for  your in the comments, fee free to refine if you want to.
All good the rest 👍 

_Note_: I wanted to propose to move it into the "Text" category, but you did it it already , good.  Let's not forget to discuss these kind of changes tho, so we can update Figma and task accordingly 😉

**Comment by anso-odoo** [comment]
@stefanorigano Thanks for the copy! Is it good for TR now?

**Review by stefanorigano — APPROVED** [review]
Neat, LGTM, thanks 👍

**Review by SergeBayet — APPROVED** [review]
@anso-odoo Nothing to say LGTM :) Thanks.
PR queued and will be r+ after https://github.com/odoo/odoo/pull/175852

**Review by SergeBayet — APPROVED** [review]

**Comment by sobo-odoo** [comment]
@robodoo delegate=SergeBayet

**Comment by SergeBayet** [comment]
@robodoo r+

**Pull request closed without merging**